pkg/operator/ceph/object/spec.go

```diff
@@ -261,6 +261,18 @@ func configureLivenessProbe(container *v1.Container, healthCheck cephv1.BucketHe
 			// If the spec value is empty, let's use a default
 			if probe != nil {
 				container.LivenessProbe = probe
+				// In case of any empty field of probe struct, we are setting default values.
+				if probe.FailureThreshold == 0 {
+					container.LivenessProbe.FailureThreshold = 3
+				} else if probe.PeriodSeconds == 0 {
+					container.LivenessProbe.PeriodSeconds = 10
+				} else if probe.SuccessThreshold == 0 {
+					container.LivenessProbe.SuccessThreshold = 1
+				} else if probe.TimeoutSeconds == 0 {
+					container.LivenessProbe.TimeoutSeconds = 1
+				} else if probe.InitialDelaySeconds == 0 {
+					container.LivenessProbe.InitialDelaySeconds = 10
+				}
```

Comment:
How about adding a test liveness probe override in the ceph_manifests.go, for example to set a different `timeoutSeconds`? Then we will be able to see the overrides working.